assembly_instructions/quality_control/tourniquet quality inspection.md

```diff
@@ -0,0 +1,81 @@
+# Quality inspection of Glia Tourniquets
+This document is revision: **A**
+
+## Revision history
+
+|Revision|Changes from previous revision|
+|---|---|
+|A | Initial creation of document|
+
+## Purpose
+
+This document contains the quality inspection characteristics and methods of inspection. In other words, what things need to be inspected and how to inspect those things.
+
+## Inspection characteristics
+
+### IC-0010 (Sewing complete)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+All sewing must be complete. All loose threads must be removed.
+
+### IC-0020 (Round end of strap heat sealed)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+The round end of the strap must be heat sealed.
+
+![IC-0020A1 End of strap heat sealed](./quality_images/IC-0020A1.jpg)
+
+
+### IC-0030 (Rivets installed)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+Both rivets are installed and secure.
+
+![IC-0030A1 Rivets in place from top](./quality_images/IC-0030A1.jpg)
+![IC-0030A2 Rivets in place from bottom](./quality_images/IC-0030A2.jpg)
+
+### IC-0040 (Clip orientation)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+The clip is installed with the "capped" end towards the windlass and buckle and the "open" end facing away.
+
+![IC-0040A1 Clip oriented with capped ends towards windlass](./quality_images/IC-0040A1.jpg)
+
+Note: the capped ends are there to provide strength to the clip. They are NOT supposed to "cover" the windlass when it is secured. The windlass will stick out thru the open end of the clip when properly secured during application to an extremity.
+
+![IC-0040A1 Clip oriented with capped ends towards windlass and windlass clipped in](./quality_images/IC-0040A2.jpg)
+
+### IC-0050 (Measurement from back plate to windlass slot)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Length measurement|Ruler OR more accurate device|
+
+The distance between the backing plate and the windlass slot must 25mm +/- 2mm.
+
+![IC-0050A1 measurement between back plate and windlass slot](./quality_images/IC-0050A1.jpg)
+
+### IC-0060 (Measurement back plate to buckle stitching)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Length measurement|Ruler OR more accurate device|
+
+The distance between the back plate and stitching that secures the buckle must be 30mm +/- 2mm.
```

Comment:
This should be measured from the bridge on the backing plate, rather than from the face.